plugins/plugins.go

```diff
@@ -0,0 +1,64 @@
+package plugins
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"path/filepath"
+	"plugin"
+
+	"github.com/gorilla/mux"
+	"github.com/prest/prest/cache"
+	"github.com/prest/prest/config"
+)
+
+// loadFunc private func to load and exec OS Library
+func loadFunc(fileName, funcName string, r *http.Request) (ret string, err error) {
+	p, err := plugin.Open(filepath.Join(config.PrestConf.PluginPath, fmt.Sprintf("%s.so", fileName)))
+	if err != nil {
+		return
```

Comment:
I'm not a Go expert but shouldn't those "return" actually return the "(ret string, err error)" ??

Reply:
`func loadFunc(fileName, funcName string, r *http.Request) (ret string, err error) {`

The function signature is using named returns, so the return is empty, returning the last declaration of the named variables

`(ret string, err error)`